### DISCUSSION TITLE
Is there a way to compile squants to be used in a Node.js project?

### DISCUSSION THREAD

**Discussion by juliobetta** [typelevel/squants#473] [Q&A]
I'm working in a Node.js project that interacts with a local Scala API, which handles unit conversions internally.  In order to keep consistency, I'm wondering if there's way to use squants directly in the JS code in case it's necessary to convert units from another sources, such as a 3rd party APIs.

**Comment by cquiroz** [comment]
Squants is published to scala.js so you could in principle use in node though maybe not that ergonomic, you may want to make a friendlier facade on scala.js
